lib/src/utils.dart

```diff
@@ -8,7 +8,7 @@ import 'dart:math';
 import 'package:rohd/rohd.dart';
 
 /// Computes the bit width needed to store [w] addresses.
-int log2Ceil(int w) => (log(w) / log(2)).ceil();
+int log2Ceil(int w) => (w > 0) ? (log(w) / log(2)).ceil() : 1;
```

Comment:
why is it good to have 1 for negative numbers?

Reply:
I reverted to the old code.  It must have been some bug somewhere that was passing w=0.